OpenOversight/tests/test_functional.py

```diff
@@ -19,6 +20,20 @@ def wait_for_page_load(browser, timeout=10):
     )
 
 
+def login_admin(browser):
+    browser.get("http://localhost:5000/auth/login")
+    with wait_for_page_load(browser):
+        elem = browser.find_element_by_id("email")
+        elem.clear()
+        elem.send_keys("test@example.org")
+        elem = browser.find_element_by_id("password")
+        elem.clear()
+        elem.send_keys("testtest")
+        with wait_for_page_load(browser):
+            browser.find_element_by_id("submit").click()
+            wait_for_element(browser, By.TAG_NAME, "body")
+
+
```

Comment:
This may conflict with PR #815. If/when this happens, holler and I’ll make a quick fix.